mobile/src/controllers/LocalSettings.ts

```diff
@@ -58,6 +58,7 @@ export class LocalSettingsController implements ILocalSettingsController {
 
         this._uid = uid;
         this._current = settings.find(s => s.deviceId === DeviceId);
+        console.log('DeviceId', DeviceId)
```

Comment:
Is this left here by mistake? If so, it should be removed (otherwise let me know why it's left here)